src/Reflection/ReflectionClass.php

```diff
@@ -160,7 +162,22 @@ public function getShortName(): string
             $fileName = sha1($this->locatedSource->getSource());
         }
 
-        return sprintf('%s%c%s(%d)', self::ANONYMOUS_CLASS_NAME_PREFIX, "\0", $fileName, $this->getStartLine());
+        return sprintf('%s%s%c%s(%d)', $this->getAnonymousClassNamePrefix(), self::ANONYMOUS_CLASS_NAME_SUFFIX, "\0", $fileName, $this->getStartLine());
+    }
+
+    private function getAnonymousClassNamePrefix(): string
```

Comment:
Let's add a couple lines explaining what this is for: it's a bit cryptic

Reply:
https://github.com/kukulich/BetterReflection/commit/07306ea292134e92d63e901f4684482da089a243